docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -25,6 +25,7 @@ Key Features and Updates
   * FIX-#4481: Allow clipping with a Modin Series of bounds (#4486)  
 * Performance enhancements
   * FEAT-#4320: Add connectorx as an alternative engine for read_sql (#4346)
+  * PERF-#4493: Use partition size caches in _prepare_frame_to_broadcast (#4495)
```

Comment:
The PR title should match the release note.

Reply:
Done.